test-support/helpers/ember-test-with-data/find-with-data.js

```diff
@@ -2,9 +2,9 @@ import Ember from 'ember';
 
 const { Test: { registerHelper } } = Ember;
 
-export default registerHelper('findWithData', (app, value = '') => {
+export default registerHelper('findWithData', (app, value = '', context=null) => {
```

Comment:
Changed the default value to be `undefined`. The `context` parameter must have a default value as it comes after an optional parameter. The value `undefined` does not break anything.